src/attributes/attr.js

```diff
@@ -114,19 +114,21 @@ boolHook = {
 };
 
 jQuery.each( jQuery.expr.match.bool.source.match( /\w+/g ), function( i, name ) {
-	var getter = attrHandle[ name ] || jQuery.find.attr;
+	var getter = attrHandle[ name.toLowerCase() ] || jQuery.find.attr;
+
+	attrHandle[ name.toLowerCase() ] = function( elem, name, isXML ) {
+		var ret, handle,
+			lowercaseName = name.toLowerCase();
```

Comment:
Where earlier? That's the beginning of the function, this name is different
than the one before its definition.

-- 
Michał Gołębiowski